Dockerfile.ci

```diff
@@ -954,7 +954,7 @@ declare -a SELECTED_TESTS CLI_TESTS API_TESTS PROVIDERS_TESTS CORE_TESTS WWW_TES
 
 function find_all_other_tests() {
     local all_tests_dirs
-    all_tests_dirs=$(find "tests" -type d ! -name '__pycache__')
+    all_tests_dirs=$(find "tests" -type d ! -name '__pycache__' | sort)
```

Comment:
nit: Would a comment be worth it to explain why we  sort the list of dirs?

Reply:
Yeah, why not. It's so obscure and hard-to-figure-out why it would make a difference that it might make sense.

I was just aiming at the smallest-possible change here, to get an example how such a small change might have an impact in a completely different place.

Reply:
Added.